tlslite/tlsconnection.py

```diff
@@ -1660,6 +1702,25 @@ def _serverGetClientHello(self, settings, certChain, verifierDB,
                                     "protocol supported"):
                                 yield result
 
+                heartbeatExt = clientHello.getExtension(
+                    ExtensionType.heartbeat)
+                if heartbeatExt:
+                    if heartbeatExt.mode == HeartbeatMode.peer_allowed_to_send:
+                        self.heartbeatCanSend = True
+                    elif heartbeatExt.mode == \
+                            HeartbeatMode.peer_not_allowed_to_send:
+                        self.heartbeatCanSend = False
+                    else:
+                        for result in self._sendError(
+                                AlertDescription.illegal_parameter,
+                                "Client sent invalid Heartbeat extension"):
+                            yield result
+                    heartbeat = HeartbeatExtension().create(
+                        HeartbeatMode.peer_allowed_to_send)
```

Comment:
What do you mean with it? Should I create some attribute in `HeartbeatExtension` class and `Heartbeat` class will work with it?